abb_irb6640_support/package.xml

```diff
@@ -0,0 +1,48 @@
+<package>
+  <name>abb_irb6640_support</name>
+  <version>1.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 6640 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 6640 manipulators. This currently includes the
+      IRB 6640-185/2.8m (6640-185) only.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in the
+      <a href="http://www.abbrobots.co.uk/en/3HAC028284-en.pdfgit s">ABB IRB 6640
+      technical data sheet</a> (Version: 3HAC 028284-001 Rev. N). All urdfs /
+      xacros are based on the default motion and joint velocity limits, unless
+      noted otherwise (ie: no support for high speed joints, extended / limited
+      motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="ds____on@swri.org">Dan Solomon (Southwest Research Institute)</author>
```

Comment:
Should this email just be removed? The `author` can stay, but the email should be removed.